content/platforms/kubernetes/getting-started/quick-start.md

```diff
@@ -0,0 +1,225 @@
+---
+Title: Quick Start for Kubernetes
+description: The Redis Enterprise Kubernetes Operator 
+  must be installed on your cluster to Redis Enterprise cluster deployments.
+weight: 10
+alwaysopen: false
+categories: ["Platforms"]
+aliases:
+---
+
+To deploy Redis Enterprise Software on Kubernetes, you first need to install the Redis Enterprise Operator. The best way to install the operator depends on your Kubernetes environment.
+
+If you're running either OpenShift or VMWare Tanzu, we provide specific getting started guides for installing the Redis Enterprise Operator on these platforms:
+
+* [Redis Enterprise on OpenShift]({{< relref "/platforms/kubernetes/getting-started/openshift/_index.md" >}})
+* [Redis Enterprise on VMWare Tanzu]({{< relref "/platforms/kubernetes/getting-started/tanzu/_index.md" >}})
+
+For all other Kubernetes deployments, continue reading this guide.
+
+## Operator overview {#overview}
+
+The Redis Enterprise Operator acts on [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs) that describe Redis Enterprise clusters and databases.
+
+The operator itself is a deployment, backed by pods, that runs within your deployment namespace. These operator pods must run with sufficient privileges to create the Redis Enterprise cluster resources within that namespace.
+
+When the operator is installed, the following resources are created:
+
+ * a service account under which the operator will run
+ * a set of Roles that define the privileges necessary for the operator to perform its tasks
+ * a set of Role Bindings to the service account to authorize the service account to have the correct roles (see above)
+ * the CRD for a Redis Enterprise Cluster
+ * the CRD for a Redis Enterprise Database
+ * the operator itself (a deployment)
+
+The operator currently runs within a single namespace and is scoped to operate only on the Redis Enterprise Cluster CRDs in that namespace.
+
+This lets you deploy multiple instances of the operator on the same Kubernetes cluster, supporting multiple Redis Enterprise Clusters in multiple namespaces.
+
+### Namespaces
+
+The Redis Enterprise Operator manages a single Redis Enterprise cluster in a single namespace.
+
+While the operator can watch additional namespaces for consumer namespaces, the Redis Enterprise Operator
+and cluster must exist in the same namespace.
```

Comment:
I don't understand this sentence. Can you confirm that it's correctly worded and necessary? @alexmilowski @amiramm

Reply:
Let's remove it. There isn't enough context in this guide about consumer namespaces and we need to keep it simple as this should be a simple guide followed almost to the letter to get started.